src/components/ThirdPartyCollectionDetailPage/CollectionItem/CollectionItem.tsx

```diff
@@ -11,8 +11,14 @@ import { getBodyShapeType } from 'modules/item/utils'
 import ItemImage from 'components/ItemImage'
 import { Props } from './CollectionItem.types'
 import * as styles from './CollectionItem.module.css'
+import ConfirmDelete from 'components/ConfirmDelete'
```

Comment:
Would you mind moving the import up a bit?

Reply:
❤️